viz_scripts/plots.py

```diff
@@ -97,22 +76,8 @@ def func(pct, values):
     plt.show()
 
 def pie_chart_purpose(plot_title,labels,values,file_name):
-    labels_trip= ['Work', 
-                  'Home',
-                  'Meal',
-                  'Shopping',
-                  'Personal/Medical',
-                  'Recreation/Exercise', 
-                  'Transit transfer', 
-                  'Pick-up/Drop off',
-                  'Entertainment/Social',
-                  'Other',
-                  'School',
-                  'Religious',
-                  'No travel', 
-                  'not_a_trip']
     
```

Comment:
Currently, I am using the labels retrieved from expanded_ct[Mode_confirm] - which is obtained from mapping in scaffolding.py. 
But as per the recent discussion to keep the hardcode color mapping [Discussion: Hardcode color mapping for individual dashboard](https://github.com/e-mission/e-mission-phone/pull/1018#issuecomment-1710945216). Should I revert my changes to keep the list of labels_trip?

Reply:
1. These are the purposes, so it is the mapping from `expanded_ct[Purpose_confirm]`.
2. I would like incremental progress on the PRs instead of waiting multiple months for a perfect PR, especially around functionality, like this, that is currently broken. As long as we know and have documented the clean up required for later, we should get the changes in now (see https://github.com/e-mission/e-mission-phone/pull/1009 and https://github.com/e-mission/e-mission-docs/issues/946 for an example of merging improvements now and polishing later)

Reply:
I have pushed my changes in the new commit.